backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -56,19 +56,19 @@ export class AdminResolver {
       searchText,
       currentPage = 1,
       pageSize = 25,
-      notActivated = null,
-      isDeleted = null,
+      filterByActivated = null,
+      filterByDeleted = null,
     }: SearchUsersArgs,
   ): Promise<SearchUsersResult> {
     const userRepository = getCustomRepository(UserRepository)
 
     const filterCriteria: ObjectLiteral[] = []
-    if (notActivated !== null) {
-      filterCriteria.push({ emailChecked: !notActivated })
+    if (filterByActivated !== null) {
+      filterCriteria.push({ filterByActivated: filterByActivated })
```

Comment:
i doubt that this works

Reply:
In my eyes it works.
Can you please explain why you think it will not work?! @ulfgebhardt

Reply:
Now I understand that I accidently renamed the database property.
Sorry @ulfgebhardt !